Mercury-Store/Mercury-Store/Home/ViewModels/HomeViewModel.swift

```diff
@@ -6,8 +6,15 @@
 //
 
 import RxSwift
+protocol HomeFlow: AnyObject {
+    func goToCategoriesTab(with itemName: String)
```

Comment:
It's strange to use name for items, it's common to use ID

Reply:
yea but it is a proof of concept on how you can transfer data with the coordinator